src/main/kotlin/com/ritense/iko/profile/ProfileRouteBuilder.kt

```diff
@@ -50,8 +53,14 @@ class ProfileRouteBuilder(private val camelContext: CamelContext, private val pr
     override fun configure() {
         val relations = profile.relations.filter { it.sourceId == null }
 
+        onException(AccessDeniedException::class.java)
+            .handled(true)
+            .setHeader(Exchange.HTTP_RESPONSE_CODE, constant(HttpStatus.UNAUTHORIZED.value()))
+
         from("direct:profile_${profile.id}")
             .routeId("profile_${profile.id}_direct")
+            .setVariable("authorities", constant("ROLE_PROFILE_${profile.name.replace(Regex("[^0-9a-zA-Z]+"), "_").uppercase()}"))
```

Comment:
I understand but it is kinda basic now should we not put a TODO here ? what is you opion here? I think TODO really help for me.